api/tests/core/booking_providers/cds/test_client.py

```diff
@@ -146,16 +152,22 @@ def test_should_raise_exception_if_show_not_found(self, mocked_get_resource):
 
 class CineDigitalServiceGetShowsRemainingPlacesTest:
     @patch("pcapi.core.booking_providers.cds.client.get_resource")
-    def test_should_return_shows_id_with_corresponding_remaining_places(self, mocked_get_resource):
+    @patch("pcapi.core.booking_providers.cds.client.CineDigitalServiceAPI.get_internet_sale_gauge_active")
```

Comment:
```suggestion
    @patch("pcapi.core.booking_providers.cds.client.CineDigitalServiceAPI.get_internet_sale_gauge_active", return_value=True)
```

Astuce: On peut définit le retour directement dans le décorateur.

Reply:
👍